x-pack/plugins/canvas/tasks/helpers/webpack.plugins.js

```diff
@@ -94,6 +94,16 @@ export function getWebpackConfig({ devtool, watch } = {}) {
           ignore: '**/__tests__/**',
         },
       ]),
+      function canvasStatsGenerator() {
+        if (!process.env.CANVAS_GENERATE_STATS) {
```

Comment:
Can you add something about this now `CANVAS_GENERATE_STATS` env var to the readme too?

Reply:
I'm going to hold off adding to the README.  There appear to be a few things out-dated... so maybe we'll double-check it for 6.7, and leave this as an undocumented feature for 6.6